README.md

```diff
@@ -1,2 +1,47 @@
 # rlistings
-Value formatting and ASCII rendering infrastructure for tables and listings.
+
+[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
```

Comment:
The rlisting package is not in a stable usable state (ie production). Please change this to whatever the official terminology is for alpha/PoC/similar is

Reply:
Changed to WIP. All status options are listed [here](https://www.repostatus.org/).